src/kartograf/atom_mapper.py

```diff
@@ -99,7 +99,7 @@ def __init__(
             with this optional parameter you can further filter the distance
             based mappings with your own custom filters, provided as iterables.
             as default we suggest to avoid ring size/breaking changes and only
-            allow whole rings to be mapped
+            allow whole rings to be mapped. These filters will be applied before any defaults
```

Comment:
I would argue that to ease usability, you want the flags to cover the best practice cases and then everything else to be optional. Thinking about CLI-based settings, it gets much harder to type the names of the filter functions rather than setting some default flags to true and false.